helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -472,11 +472,15 @@ public void updateDeploymentGroupHosts(final String name, final List<String> hos
           }
         }
 
+        log.info("starting zookeeper transaction for updateDeploymentGroupHosts on "
```

Comment:
I still find it somewhat confusing that these are logged before the transaction is successfully committed. This is intended to be racey so the transaction will fail on occasion.